apps/cli/src/commands/rollups/start.ts

```diff
@@ -0,0 +1,196 @@
+import { Command, Option } from "@commander-js/extra-typings";
+import chalk from "chalk";
+import { execa } from "execa";
+import path from "path";
+
+const commaSeparatedList = (value: string, _previous: string[]) =>
+    value.split(",");
+
+const availableServices = [
+    "bundler",
+    "explorer",
```

Comment:
```suggestion
    "bundler",
    "espresso",
    "explorer",
```

Reply:
If we agree with `--sequencer espresso`, then `espresso` would not be enabled by `--services`, so it should not be part of this list.